src/CachePoolTest.php

```diff
@@ -106,6 +106,20 @@ public function testBasicUsage()
         $this->assertFalse($this->cache->getItem('key2')->isHit());
     }
 
+    public function testItemModifiersReturnsStatic()
+    {
+        if (isset($this->skippedTests[__FUNCTION__])) {
+            $this->markTestSkipped($this->skippedTests[__FUNCTION__]);
+
+            return;
+        }
+
+        $item = $this->cache->getItem('key');
+        $this->assertSame($item, $item->set('4711'));
+        $this->assertSame($item, $item->expiresAfter(2));
+        $this->assertSame($item, $item->expiredAt(new \DateTime('+2hours')));
```

Comment:
Sorry, I missed that while reviewing. 

This test just want to see if `expiresAfter` and `expiresAt` returns `$this`. You are correct, we do test expiration in a different test: https://github.com/php-cache/integration-tests/blob/master/src/CachePoolTest.php#L406